gradle.properties

```diff
@@ -1,6 +1,6 @@
 #Fri Mar 06 02:45:42 UTC 2020
 enablePublishing=false
-spinnakerGradleVersion=7.1.1
+spinnakerGradleVersion=7.7.0
```

Comment:
@cfieber You reverted the last bump to both `spinnakerGradleVersion` and the project Gradle version.  Do you know if that publishing issue was due to the Gradle version bump or this dependency bump?

Reply:
it was due to me trying to get us onto gradle 6.x at the same time, which didn't fly

Reply:
if you can run a `publishToMavenLocal` I think everything should be happy